frc.go

```diff
@@ -142,21 +142,21 @@ func ListInfo(header string, titles []string, data []interface{}) {
 	fmt.Println()
 }
 
-func DisplayTeam(team tbago.Team) {
+func displayTeam(team tbago.Team) {
 	header := fmt.Sprintf("Team %d", team.TeamNumber)
 	titles := []string{"Nickname", "Website", "Rookie", "Country", "Motto"}
 	data := []interface{}{team.Nickname, team.Website, team.RookieYear, team.Country, team.Motto}
-	ListInfo(header, titles, data)
+	listInfo(header, titles, data)
 }
 
-func DisplayEvent(event tbago.Event) {
+func displayEvent(event tbago.Event) {
```

Comment:
Is this sort of camelCase conventional for unexported functions?

Reply:
When a function name starts with an uppercase character it is exported

Reply:
https://tour.golang.org/basics/3